codegen/reconcile/main.go

```diff
@@ -166,6 +166,11 @@ func main() {
 				ImportAlias:      "kubermaticv1",
 				APIVersionPrefix: "KubermaticV1",
 			},
+			{
+				ResourceName:     "Project",
+				ImportAlias:      "kubermaticv1",
+				APIVersionPrefix: "KubermaticV1",
```

Comment:
Dont think you need this prefix. I needed it because there were 2 kinds of ConstraintTemplates. 

On the other hand, it doesnt hurt, and also could help if we create a project V2 in the future. So IMO it can stay

Reply:
Yeah, I am also thinking that we can handle multiple versions of `Project` in the future, not sure which is better for now :)